### PR TITLE
removal(note): remove terra-note

### PR DESCRIPTION
**Pull request by nappynapster**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
-# 12.0.0-next.0 (13.07.2021)
-
-### Breaking Changes
+# X.X.X (XX.07.2021)
```
Comment (pweyrich) [inline CHANGELOG.md:1]: Bitte die Version angeben, die deine Änderungen nach sich zieht.
```suggestion
# 12.0.0-next.1 (XX.07.2021)
```

**Review by pweyrich — DISMISSED** [review]

**Comment by niwolf** [comment]
In `src/lib/components/editors/note-editor/example/terra-note-editor.component.example.html` muss das noch raus

**Comment by nappynapster** [comment]
> In `src/lib/components/editors/note-editor/example/terra-note-editor.component.example.html` muss das noch raus

ne das leider von dem terra-note-editor. Der basiert auf der quill lib

**Comment by pweyrich** [comment]
> > In `src/lib/components/editors/note-editor/example/terra-note-editor.component.example.html` muss das noch raus
> 
> ne das leider von dem terra-note-editor. Der basiert auf der quill lib

Niklas meint, dass es in diesem example noch eine Verwendung von der note component gibt. Die muss raus.
Deshalb schlägt der build auch noch fehl.. (siehe https://github.com/plentymarkets/terra-components/pull/1608/checks?check_run_id=3057936560)
Bildschirmfoto 2021-07-14 um 09 06 35

**Review by pweyrich — APPROVED** [review]

**Review by zhajjhassan — APPROVED** [review]

**Pull request merged**